lib/util.js

```diff
@@ -204,6 +204,12 @@ function pad(n) {
  * @returns {string}
  */
 function styleText(format, text) {
+  const env = process.env;
+  if (!process.stdout.isTTY ||
```

Comment:
I'm not sure it's the right thing to systematically look at `process.stdout`. `styleText`, as an utility function, should not (always) depend on that.

Reply:
When using such a utility, you aren't necessarily writing on to stdout. You might be writing to a file, and you might want to use coloring even without a tty

Reply:
That's a fair point.

Reply:
What about adding a boolean parameter to the function set to `true` by default to check the TTY: `styleText(format, text, validateTTY)`? So we offer the option to use the function with or without the proposed validation?

The idea behind this validation is to provide the mechanism by default to Node users, saving them time with some code that would be a must in the most common use case of this utility function.

I will be waiting for your feedback, @targos and @MoLow to implement such suggestion

Reply:
SGTM